### PR TITLE
python-openstackclient: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/python-openstackclient/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3b11c423a38074b0b58ba66d5f6e5334759f50e99fe19c3dfc662a6299dc2e16
+
+build:
+  number: 0
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/python-openstackclient/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/python-openstackclient/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+test:
+  imports:
+    - openstackclient
+  commands:
+    - openstack --help
+
+
+about:
+  home: https://docs.openstack.org/python-openstackclient/latest/
+  license: "Apache Software"
```
Comment (marcelotrevisani) [inline recipes/python-openstackclient/meta.yaml]:
```suggestion
  license: "Apache-2.0"
```

**Comment by tschoonj** [comment]
@conda-forge/staged-recipes 

Can somebody have a look at my [warlock-feedstock PR](https://github.com/conda-forge/warlock-feedstock/pull/6)? It appears that this repo is no longer maintained and I want to step up and help out.

The warlock update is kind of import for this recipe since without it, `python-openstackclient` cannot be installed on python > 3.5 :cry: 

Thanks in advance

**Comment by scopatz** [comment]
Is this OK to go in as-is?

**Comment by tschoonj** [comment]
It is now!

Many thanks again!

**Pull request merged**